src/netconf_acm.c

```diff
@@ -0,0 +1,1862 @@
+/**
+ * @file netconf_acm.c
+ * @author Michal Vasko <mvasko@cesnet.cz>
+ * @brief NACM and ietf-netconf-acm callbacks
+ *
+ * @copyright
+ * Copyright (c) 2019 - 2021 Deutsche Telekom AG.
+ * Copyright (c) 2017 - 2021 CESNET, z.s.p.o.
+ *
+ * This source code is licensed under BSD 3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://opensource.org/licenses/BSD-3-Clause
+ */
+#define _GNU_SOURCE
+#define _DEFAULT_SOURCE
```

Comment:
Please remove this, it should not be possible and using `_GNU_SOURCE` should be always enough.

Reply:
Done